src/startupscriptgenerator/src/dotnetcore/scriptgenerator.go

```diff
@@ -81,6 +113,15 @@ func (gen *DotnetCoreStartupScriptGenerator) GenerateEntrypointScript(scriptBuil
 	common.SetEnvironmentVariableInScript(scriptBuilder, "PORT", gen.BindPort, DefaultBindPort)
 	scriptBuilder.WriteString("export ASPNETCORE_URLS=http://*:$PORT\n\n")
 
+	logger.LogInformation("Setting up Kestrel Endpoints with BindPort and BindPort2 Env variables if .NET runtime version >= 7")
```

Comment:
I actually don't believe we need to restrict it to .NET 7. I believe the info in the email was to limit it to .NET 7 if it was easier, but that fundamentally we wanted it for all .NET versions. Could you please clarify: is there any way to limit it to .NET 7?

Reply:
Yes we're actually limited to .NET 7 with the function at line 118. My understanding from the email thread they're asking to scope to .NET 7 first, then .NET 6.

`if gen.isDotnetRuntimeVersionMeetConstraint("7.0.0-0") {`

Reply:
Or we should remove this line? Please let me know if I understand it wrongly.

Reply:
Yes, it's ambiguous. Jenny and I mentioned that we should support all .NET versions if it's possible, but Surender mentioned that having only .NET 7 would be better to reduce the blast radius.

Let's keep what you have, and we can remove the line as needed in the future.